exchanges/coinbasepro/coinbasepro_types.go

```diff
@@ -93,68 +95,68 @@ type AccountLedgerResponse struct {
+	CreatedAt time.Time `json:"created_at"`
+	UpdatedAt string    `json:"updated_at"`
+	Amount    float64   `json:"amount,string"`
+	Type      string    `json:"type"`
+	Reference string    `json:"ref"`
 }
 
 // GeneralizedOrderResponse is the generalized return type across order
 // placement and information collation
 type GeneralizedOrderResponse struct {
-	ID             string  `json:"id"`
-	Price          float64 `json:"price,string"`
-	Size           float64 `json:"size,string"`
-	ProductID      string  `json:"product_id"`
-	Side           string  `json:"side"`
-	Stp            string  `json:"stp"`
-	Type           string  `json:"type"`
-	TimeInForce    string  `json:"time_in_force"`
-	PostOnly       bool    `json:"post_only"`
-	CreatedAt      string  `json:"created_at"`
-	FillFees       float64 `json:"fill_fees,string"`
-	FilledSize     float64 `json:"filled_size,string"`
-	ExecutedValue  float64 `json:"executed_value,string"`
-	Status         string  `json:"status"`
-	Settled        bool    `json:"settled"`
-	Funds          float64 `json:"funds,string"`
-	SpecifiedFunds float64 `json:"specified_funds,string"`
-	DoneReason     string  `json:"done_reason"`
-	DoneAt         string  `json:"done_at"`
+	ID             string    `json:"id"`
+	Price          float64   `json:"price,string"`
+	Size           float64   `json:"size,string"`
+	ProductID      string    `json:"product_id"`
+	Side           string    `json:"side"`
+	Stp            string    `json:"stp"`
+	Type           string    `json:"type"`
+	TimeInForce    string    `json:"time_in_force"`
+	PostOnly       bool      `json:"post_only"`
+	CreatedAt      time.Time `json:"created_at"`
+	FillFees       float64   `json:"fill_fees,string"`
+	FilledSize     float64   `json:"filled_size,string"`
+	ExecutedValue  float64   `json:"executed_value,string"`
+	Status         string    `json:"status"`
+	Settled        bool      `json:"settled"`
+	Funds          float64   `json:"funds,string"`
+	SpecifiedFunds float64   `json:"specified_funds,string"`
+	DoneReason     string    `json:"done_reason"`
+	DoneAt         string    `json:"done_at"`
 }
 
 // Funding holds funding data
 type Funding struct {
-	ID            string  `json:"id"`
-	OrderID       string  `json:"order_id"`
-	ProfileID     string  `json:"profile_id"`
-	Amount        float64 `json:"amount,string"`
-	Status        string  `json:"status"`
-	CreatedAt     string  `json:"created_at"`
-	Currency      string  `json:"currency"`
-	RepaidAmount  float64 `json:"repaid_amount"`
-	DefaultAmount float64 `json:"default_amount,string"`
-	RepaidDefault bool    `json:"repaid_default"`
+	ID            string    `json:"id"`
+	OrderID       string    `json:"order_id"`
+	ProfileID     string    `json:"profile_id"`
+	Amount        float64   `json:"amount,string"`
+	Status        string    `json:"status"`
+	CreatedAt     time.Time `json:"created_at"`
+	Currency      string    `json:"currency"`
+	RepaidAmount  float64   `json:"repaid_amount"`
```

Comment:
dont have history on the exchange to check but since all other 'amounts' are using ,string maybe this one might require it too?

Reply:
I'm not even seeing this `funding` endpoint in https://docs.pro.coinbase.com/